files/lib/system/exporter/WBB4xExporter.class.php

```diff
@@ -2347,12 +2347,17 @@ public function countFilebaseFiles() {
 	}
 	
 	/**
-	 * Exports filebase files.
+	 * Exports filebase files from 5.2-.
```

Comment:
While I consider a `+` to be fairly clear the `-` looks like a typo. I suggest using an explicit `5.2 or lower` here.